src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -121,8 +121,8 @@ public class Settings {
     @YamlComment("Ordered list of elements by which players should be sorted. " +
             "(ROLE_WEIGHT, ROLE_NAME, SERVER_NAME, SERVER_GROUP and SERVER_GROUP_NAME are supported)")
     private List<String> sortPlayersBy = List.of(
```

Comment:
Let's change the `@YamlKey` of this and variable name to `sortingPlaceholders` to prevent issues when updating without a config regen and correct the `@YamlComment` to be more accurate.

Also, could you update the corresponding docs?